frontend/components.js

```diff
@@ -85,25 +320,68 @@ class UnlockClass extends Component {
             },
             'Try demo wallet'
           )
+        )
+      )
+
+    const LoadByHardwareWalletSection = () =>
+      h(
+        'div',
+        {class: 'auth-section'},
+        h(
+          'div',
+          undefined,
+          'Hardware wallets porvide the best security level for your Cryptocurrency.'
```

Comment:
typo: "porvide" -> "provide" I would reword it like this: "Hardware wallets provide the best security level for storing your cryptocurrencies."